### PR TITLE
Bump to 2.25.0

### PR DESCRIPTION
**Pull request by PettitWesley**
Signed-off-by: Wesley Pettit <wppttt@amazon.com>

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,16 @@
...
+This release includes:
+* An Amazon Linux 2 Base
+* Fluent Bit [1.9.3](https://fluentbit.io/announcements/v1.9.3/)
+* Amazon CloudWatch Logs for Fluent Bit 1.7.0
+* Amazon Kinesis Streams for Fluent Bit 1.9.0
+* Amazon Kinesis Firehose for Fluent Bit 1.6.1
+
+Compared to `2.23.4` this release adds the following feature that we are working on getting accepted upstream:
+* Enhancement - Add `kube_token_ttl` option to kubernetes filter to support refreshing the service account token used to talk to the API server. Prior to this change Fluent Bit would only read the token on startup. [fluentbit:5332](https://github.com/fluent/fluent-bit/issues/5332)
+
```
Comment (matthewfala) [inline CHANGELOG.md:13]: Should we update the 2.24.0 changelog to reflect the changes in our release notes?

**Review by matthewfala — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (matthewfala) [reply CHANGELOG.md:13]: https://github.com/aws/aws-for-fluent-bit/releases/tag/v2.24.0

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,16 @@
...
 
+### 2.25.0
+This release includes:
+* An Amazon Linux 2 Base
+* Fluent Bit [1.9.3](https://fluentbit.io/announcements/v1.9.3/)
+* Amazon CloudWatch Logs for Fluent Bit 1.7.0
+* Amazon Kinesis Streams for Fluent Bit 1.9.0
+* Amazon Kinesis Firehose for Fluent Bit 1.6.1
+
+Compared to `2.23.4` this release adds the following feature that we are working on getting accepted upstream:
```
Comment (zhonghui12) [inline CHANGELOG.md:11]: we should say `2.24.0` here?

File: CHANGELOG.md (hunk 1 above)
Comment (PettitWesley) [reply CHANGELOG.md:13]: Can you submit a separate PR for that? Its not really logically part of this version bump.

File: CHANGELOG.md (hunk 2 above)
Comment (PettitWesley) [reply CHANGELOG.md:11]: Oops/Thanks

**Pull request merged**